app/services/reports/monthly_usps_letter_requests_report.rb

```diff
@@ -4,9 +4,13 @@ module Reports
   class MonthlyUspsLetterRequestsReport < BaseReport
     REPORT_NAME = 'monthly-usps-letter-requests-report'.freeze
 
-    def call
+    def call(start_time: nil, end_time: nil)
+      start_time ||= first_of_this_month
+
+      end_time ||= end_of_today
```

Comment:
why not just have those be the default arguments? what are the cases we'd want to pass explicit nil?

```suggestion
    def call(start_time: first_of_this_month, end_time: end_of_today)
```

Reply:
I didn't know that was a thing 🙂